practice-app/back-end/routes/twittercomments.js

```diff
@@ -6,14 +6,15 @@ router.get('/', function (req, res, next) {
   // import twitter
   var Twitter = require('twitter');
   //get the all access keys
-  var key = require('./key.json');
+  var key = require('./keys.js');
   //create new twitter object
   var client = new Twitter(key);
-
+  console.log("count is", req.query.count);
```

Comment:
delete this :D or change to full log : so log the query.query. too

Reply:
Aye aye sir.